doc/integrator/filter.rst

```diff
@@ -3,13 +3,53 @@
 Filter (Querier)
 ================
 
+Filterable layers
+-----------------
+Per default, layers are not filterable.
+If you wish to provide the filter functionality, proceed as follows:
+
+* ensure that the functionality ``filterable_layers`` is made available in your project
+  template files:
+
+  - if you have made the default functionalities of ``CONST_vars.yaml`` available in your
```

Comment:
That's more like:
If you have a custom definition for the `functionalities.available_in_templates`, or `admin_interface.available_functionalities` in your vars.yaml file then ensure that the value `filterable_layers` is defined under both variables. Alternatively you can add these two paths in the `update_paths` variable to update the CONST_Vars definition of these two variables (and keep the inheritance) instead of replace them.

(Probably a poor English, but that's conceptually more right I think.)

Reply:
Thanks I have integrated that in a new proposal - what do you think?